Teatoller/laravel-basics
# #002  Implement Project Setup

#### What does this PR do?
- Project set up
#### Description of Task to be completed?
- [x] set up the project file and add dependencies
- [x] add Foundation-5 files
- [x] add routes
- [x] add layout file
- [x] render extended view
#### How should this be manually tested?
1. Create a folder on your machine `mkdir myproject`
2. `cd myproject`
3. Run `git clone https://github.com/Teatoller/laravel-basics.git`
4. cd `laravel-basics`
5. open **laravel-basics** in your chosen editor. For **vscode** enter `code .` on terminal and press enter.
6. `git checkout ft-project-setup-001`
7. Run `composer install`
8. Run `php artisan serve`  the app in the development mode to open [ http://127.0.0.1:8000/](http://127.0.0.1:8000/) to view it in the browser.
9. Navigate to **endpoint** : http://127.0.0.1:8000/hello to see view as in  **screenshot: 1**

#### Any background context you want to provide?
- n/a
#### What are the relevant pivotal tracker stories?
- n/a
#### Screenshots (if appropriate)
**Screenshot:1**
![setup](https://user-images.githubusercontent.com/39020475/69740226-1cffce00-114a-11ea-86e6-2df01b1db54b.png)

#### Questions:
